assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.scss

```diff
@@ -1,22 +1,20 @@
 .component-stripe-pop-up-button {
-  background-color: gu-colour(neutral-1);
+  box-sizing: border-box;
```

Comment:
Since the css for the StripePopupButton is now the same on the monthly contributions payment page and the one off contribution payment page and it doesn't seem to be used elsewhere, I figured why not move it here.

Reply:
I think we are using a Stripe pop up button in the [regular checkout](https://github.com/guardian/support-frontend/blob/master/assets/pages/regular-contributions/regularContributions.scss#L129). So this styles will be added to that page, which before didn't have those.